compiler/AST/AggregateType.cpp

```diff
@@ -1854,6 +1855,14 @@ void AggregateType::fieldToArg(FnSymbol*              fn,
           arg->addFlag(FLAG_TYPE_VARIABLE);
         }
 
+        if (LoopExpr* fe = toLoopExpr(defPoint->init)) {
+          if (field->isType() == false) {
```

Comment:
I have a mild preference for ``foo == false`` over ``!foo``, because sometimes my eyes will too quickly scan past the ``!``. But I don't feel that strongly about it.